packages/flutter_tools/test/general.shard/commands/analyze_once_test.dart

```diff
@@ -99,7 +99,7 @@ void main() {
         statusTextContains: <String>[
           'Analyzing',
           'warning $analyzerSeparator The parameter \'onPressed\' is required',
-          'info $analyzerSeparator The method \'_incrementCounter\' isn\'t used',
```

Comment:
Unfortunately that was not the only change. The word `method` was replaced with word `declaration` as well.
```
   Expected: contains 'info • The method \'_incrementCounter\' isn\'t used'
  Actual: 'Analyzing flutter_project...\n'
            '\n'
            '   info • The declaration \'_incrementCounter\' isn\'t referenced • ../../../../../tmp/t/flutter_analyze_once_test_1.EEEWQC/flutter_project/lib/main.dart:49:8 • unused_element\n'
```

Reply:
[Siva's roll](https://github.com/flutter/flutter/commit/c15cfdd6af8431bd533cfbf4c195176061b0f680) included that missing change.